test/integration/visitor_can_add_items_to_cart_test.rb

```diff
@@ -33,14 +33,75 @@ class VisitorCanAddItemsToCart < ActionDispatch::IntegrationTest
   end
 
   test "can add multiple items to cart" do
-    skip
+    category_1 = Oil.create(name: "Lard")
+    category_2 = Oil.create(name: "Coconut Oil")
+
+    item_1 = Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_id: category_1.id)
+    item_2 = Chip.create(name: "Trader Joe's BBQ", price: 15, description: "I'd trade slota for these!", oil_id: category_2.id)
+    item_3 = Chip.create(name: "Dang Coconut", price: 17, description: "Dang, these are good", oil_id: category_2.id)
+    item_4 = Chip.create(name: "Lard Yummies", price: 19, description: "Chock Full of Lard", oil_id: category_1.id)
```

Comment:
Useless assignment to variable - `item_4`. Did you mean `item_1`?<br>Line is too long. [115/80]